gpu-operator/dra-driver.rst

```diff
@@ -0,0 +1,628 @@
+      [Mar 27 2025 15:47:10] [INFO] [tid 39] IMEX version 570.124.06 is running with the following configuration options
+
+      [Mar 27 2025 15:47:10] [INFO] [tid 39] Logging level = 4
+
+      [Mar 27 2025 15:47:10] [INFO] [tid 39] Logging file name/path = /var/log/nvidia-imex.log
+
+      [Mar 27 2025 15:47:10] [INFO] [tid 39] Append to log file = 0
+
+      [Mar 27 2025 15:47:10] [INFO] [tid 39] Max Log file size = 1024 (MBs)
+
+      [Mar 27 2025 15:47:10] [INFO] [tid 39] Use Syslog file = 0
+
+      [Mar 27 2025 15:47:10] [INFO] [tid 39] IMEX Library communication bind interface =
+
+      [Mar 27 2025 15:47:10] [INFO] [tid 39] IMEX library communication bind port = 50000
+
+      [Mar 27 2025 15:47:10] [INFO] [tid 39] Identified this node as ID 0, using bind IP of '10.115.131.8', and network interface of enP5p9s0
+      [Mar 27 2025 15:47:10] [INFO] [tid 39] nvidia-imex persistence file /var/run/nvidia-imex/persist.dat does not exist.  Assuming no previous importers.
+      [Mar 27 2025 15:47:10] [INFO] [tid 39] NvGpu Library version matched with GPU Driver version
+      [Mar 27 2025 15:47:10] [INFO] [tid 63] Started processing of incoming messages.
+      [Mar 27 2025 15:47:10] [INFO] [tid 64] Started processing of incoming messages.
+      [Mar 27 2025 15:47:10] [INFO] [tid 65] Started processing of incoming messages.
+      [Mar 27 2025 15:47:10] [INFO] [tid 39] Creating gRPC channels to all peers (nPeers = 1).
+      [Mar 27 2025 15:47:10] [INFO] [tid 66] Started processing of incoming messages.
+      [Mar 27 2025 15:47:10] [INFO] [tid 39] IMEX_WAIT_FOR_QUORUM != FULL, continuing initialization without waiting for connections to all nodes.
+      [Mar 27 2025 15:47:10] [INFO] [tid 67] Connection established to node 0 with ip address 10.115.131.8. Number of times connected: 1
+      [Mar 27 2025 15:47:10] [INFO] [tid 39] GPU event successfully subscribed
+
+  
+#. Delete ``imex-channel-injection`` example.
+
+   .. code-block:: console
+
+      $ kubectl delete -f imex-channel-injection.yaml
+
+   *Example Output*
+
+   .. code-block:: output
+
+      computedomain.resource.nvidia.com "imex-channel-injection" deleted
+      pod "imex-channel-injection" deleted
+
+
+******************************************************************
+Run a Multi-node nvbandwidth Test Requiring IMEX Channels with MPI
+******************************************************************
+
+This example demonstrates how to run a workload across multiple nodes using a ComputeDomain. 
+The nvbandwidth test will measure the bandwidth between GPUs across different nodes using IMEX channels, helping you verify that your MNNVL setup is working correctly.
+
+The following sample files use the following:
+
+- `Kubeflow MPI Operator <https://www.kubeflow.org/docs/components/trainer/legacy-v1/user-guides/mpi/#installationr>`__  
+- A 2 node cluster with all nodes capable of running GPU workloads and meets [NVIDIA DRA Driver for GPUs prerequisites](#prerequisites)

```

Comment:
@a-mccarthy We need to specify the number of GPUs. 
 
Please note that this example is for 2 node cluster which has 4 GPUs. Change the following according to your environment:
`spec.numNodes` | Total number of nodes in the cluster | `2` |
`spec.slotsPerWorker` | Number of GPUs per node, this should match the ppr number  | `4` |
`Worker.replicas` | Number of worker nodes | `2` |
`args` to mpirun | Adjust process-per-resource and total processes | `-ppr:4(numGPUs per node):node -np 8(numGPU x numNodes)` |

cc @ArangoGutierrez to help with formatting. Not sure whats the best way to specify these in detail. I have seen many people making mistake with these numbers so i think we should definitely add a note.

Reply:
@guptaNswati thanks! I've updated the docs to include the table, i also added some inline comments to callout how to update the example to a different cluster set up. can you take another look and make sure it looks ok?

Reply:
LGTM. Thanks @a-mccarthy